editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -237,7 +237,7 @@ describe('React Render Count Tests - ', () => {
     )
 
     const renderCountAfter = renderResult.getNumberOfRenders()
-    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(2725) // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toBeLessThan(2735)
+    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(1900) // if this breaks, GREAT NEWS but update the test please :)
```

Comment:
I -think- this happens because something changed around module resolution, and some React components doesn't see the monkey patched react that counts renders

Reply:
When this shot up previously I captured the components that were being rendered by modifying the patched React function, might be worth doing that to confirm your theory?

Reply:
#1872 fixed this a lot, it's still a slightly smaller number, but I think it's acceptable now